test/extended/node/node_swap.go

```diff
@@ -0,0 +1,180 @@
+package node
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/kubernetes/test/e2e/framework"
+
+	machineconfigv1 "github.com/openshift/api/machineconfiguration/v1"
+	mcclient "github.com/openshift/client-go/machineconfiguration/clientset/versioned"
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("[OTP][Jira:\"Node / Kubelet\"][sig-node] Node non-cnv swap configuration", func() {
```

Comment:
[OTP] means test cases that are ported(from openshift-test-private), here the case is a newly created, we don't need add it. 
[Jira:Component] track the ownership of the case, "Node" is enough, no need "Kubelet"